bciers/apps/registration/app/components/operations/registration/OperationInformationForm.tsx

```diff
@@ -1,46 +1,117 @@
 "use client";
 
 import MultiStepBase from "@bciers/components/form/MultiStepBase";
-import { operationInformationUiSchema } from "apps/registration/app/data/jsonSchema/operationRegistration/operationInformation";
 import { OperationInformationFormData } from "apps/registration/app/components/operations/registration/types";
 import { actionHandler } from "@bciers/actions";
 import { useRouter } from "next/navigation";
 import { RJSFSchema } from "@rjsf/utils";
+import { useState } from "react";
+import { IChangeEvent } from "@rjsf/core";
+import { getOperation } from "@bciers/actions/api";
+import {
+  createNestedFormData,
+  createUnnestedFormData,
+} from "@bciers/components/form/formDataUtils";
+import { registrationOperationInformationUiSchema } from "@/registration/app/data/jsonSchema/operationInformation/registrationOperationInformation";
 
 interface OperationInformationFormProps {
-  formData: OperationInformationFormData;
+  rawFormData: OperationInformationFormData;
   schema: RJSFSchema;
   step: number;
   steps: string[];
 }
 
 const OperationInformationForm = ({
-  formData,
+  rawFormData,
   schema,
   step,
   steps,
 }: OperationInformationFormProps) => {
+  const [selectedOperation, setSelectedOperation] = useState("");
+  const [, setError] = useState(undefined);
```

Comment:
we need to be storing the error in a variable, no?

Reply:
i.e., `const [error, setError] = useState(undefined);`

Reply:
We're not using the `error` variable yet (part of https://github.com/orgs/bcgov/projects/122/views/2?filterQuery=cleanup&pane=issue&itemId=75086601), so ts throws an error if I include and it's not used anywhere. I could leave it in an put in an ignore if you prefer

Reply:
ah gotcha, it's fine leaving as-is for now